parquet-common/src/main/java/org/apache/parquet/bytes/MultiBufferInputStream.java

```diff
@@ -379,4 +427,120 @@ public void remove() {
       second.remove();
     }
   }
+
+  @Override
+  public byte readByte() throws IOException {
+    return (byte)readUnsignedByte();
```

Comment:
nit: can we add a space after the cast `(byte) readUnsignedByte()`? for all the other places too

Reply:
got it